LocalPackages/DataBrokerProtection/Sources/DataBrokerProtection/Scheduler/DataBrokerProtectionQueueManager.swift

```diff
@@ -24,10 +24,14 @@ protocol DataBrokerProtectionOperationQueue {
     var maxConcurrentOperationCount: Int { get set }
     func cancelAllOperations()
     func addOperation(_ op: Operation)
-    func addBarrierBlock(_ barrier: @escaping @Sendable () -> Void)
+    func addBarrierBlock1(_ barrier: @escaping @Sendable () -> Void)
```

Comment:
I was sure we had solved it in main branch using some nicer name 😂 @THISISDINOSAUR do you recall handling this Xcode 16 compilation issue?

Reply:
We did, and then I think @quanganhdo was able to delete it. @quanganhdo I can't remember the details, can we remove this?

Reply:
We were able to delete for a particular Xcode 16 revision only. Later updates broke it again.